conf/modules.config

```diff
@@ -38,4 +35,352 @@ process {
         ]
     }
 
+    /////////////////////// PROCESS GENOME ////////////////////////////////////
+
+    // This applies to all PREPARE_GENOME steps
+    withName: ".*:PREPARE_GENOME:.*" {
+        publishDir = [
+            enabled: params.save_genome_intermediate,
+            path: { "${params.outdir}/genome/${task.process.tokenize(':')[-1].tokenize('_')[0].toLowerCase()}" },
+            mode: params.publish_dir_mode,
+            saveAs: { filename -> filename.equals('versions.yml') ? null : filename }
+        ]
+    }
+    // bwamem2_index can take a lot of memory for larger genomes. This gives
+    // the pipeline the chance to scale up to 120G for this. The enormous
+    // amount of memory should be fixed in versions of bwamem2 after ~2022 01
+    withName: BWAMEM2_INDEX {
+        maxRetries = 5
+        memory = { check_max( 24.GB * task.attempt, 'memory' ) }
+    }
```

Comment:
this should go to the base config. All resource configurations are generally stored there to keep them in one place